Common/include/grid_movement/CFreeFormDefBox.hpp

```diff
@@ -502,49 +502,49 @@ class CFreeFormDefBox : public CGridMovement {
    * \brief Set Cylindrical to Cartesians_ControlPoints.
    * \param[in] config - Definition of the particular problem.
    */
-  void SetCyl2Cart_ControlPoints(CConfig* config);
+  void SetCyl2Cart_ControlPoints(CConfig* config) const;
 
   /*!
    * \brief Set Cartesians to Cylindrical ControlPoints.
    * \param[in] config - Definition of the particular problem.
    */
-  void SetCart2Cyl_ControlPoints(CConfig* config);
+  void SetCart2Cyl_ControlPoints(CConfig* config) const;
```

Comment:
Many of these const changes are incorrect. These functions modify the class, but they do so through a member variable that is a pointer, and thus C++ allows const qualification since the pointer doesn't change, but the data does.
Note that the functions are called SetSomething, therefore there is clear intent in modifying something.

Don't get me wrong but I would suggest that you avoid automatic changes of this type until you know the code base better.

Reply:
I reverted the false ones and removed the check, almost all of them from Geometry class.

It should be fine now. Please check and merge.